tests/components/grid.test.ts

```diff
@@ -1429,18 +1432,42 @@ describe("Copy paste keyboard shortcut", () => {
     expect(getCellContent(model, "A2")).toEqual("things");
   });
 
-  test("cut zone gets cleared on paste if content/style is altered after cut", async () => {
+  test("Cannot paste if content is altered after cut", async () => {
```

Comment:
The 2 tests should probably be in `clipboard.test.ts` since we don't test anything dependant to grid